.travis.yml

```diff
@@ -5,21 +5,18 @@ language: ruby
 cache: bundler
 before_install:
   - bundle -v
-  - rm Gemfile.lock || true
+  - rm -f Gemfile.lock
   - gem update --system
   - gem update bundler
   - gem --version
   - bundle -v
 script:
-  - 'COVERAGE=yes bundle exec rake $CHECK'
+  - 'bundle exec rake $CHECK'
```

Comment:
The PDK sync kills our coverage tests

Reply:
I ended up down a PDK rabbit hole. I've pegged this back to just the immediate git get of the tagged release version.

Though I feel that re-enabling PDK should be done when I figure out how to enable coverage in travis.yml